src/server/journal/serializer.cc

```diff
@@ -71,16 +71,20 @@ void JournalWriter::Write(const journal::Entry& entry) {
     case journal::Op::SELECT:
       return Write(entry.dbid);
     case journal::Op::COMMAND:
+    case journal::Op::MULTI_COMMAND:
       Write(entry.txid);
       Write(entry.shard_cnt);
       return std::visit([this](const auto& payload) { return Write(payload); }, entry.payload);
+    case journal::Op::EXEC:
+      Write(entry.txid);
+      return Write(entry.shard_cnt);
```

Comment:
Whats the payload for in this case? `std::monostate`, i.e. nullopt? Then we can just merge it with the previous case, because writing the payload does nothing

Reply:
ok